src/servlets/SignInServlet.java

```diff
@@ -0,0 +1,41 @@
+package servlets;
+
+import templater.PageGenerator;
+import user.AccountService;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class SignInServlet extends HttpServlet {
+    private AccountService account;
+
+    public SignInServlet(AccountService account) {
+        this.account = account;
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
```

Comment:
try to use full name

Reply:
Ok. i didn't do it 'cause i thought that in this method it would be too many code's lines